src/app/history/order/[orderHash]/(components)/OrderInformation.tsx

```diff
@@ -0,0 +1,29 @@
+import { Tooltip, TooltipContent, TooltipTrigger } from "@bleu-fi/ui";
+import { QuestionMarkCircledIcon } from "@radix-ui/react-icons";
+import { TooltipProvider } from "@radix-ui/react-tooltip";
+
+export function OrderInformation({
+  label,
+  children,
+  tooltipText,
+}: {
+  label: string;
+  children: React.ReactNode;
+  tooltipText?: string;
+}) {
+  return (
+    <TooltipProvider>
+      <div className="flex items-center gap-x-2">
+        {tooltipText && (
+          <Tooltip>
+            <TooltipContent>{tooltipText}</TooltipContent>
+            <TooltipTrigger disabled>
+              <QuestionMarkCircledIcon />
+            </TooltipTrigger>
+          </Tooltip>
+        )}
```

Comment:
I think the the more aesthetic choice is for the tooltip to go AFTER the label, or, even better IMO since we're adding tooltips to everything, to have the label itself within the tooltip so that the UI does't get full of question marks 😅 .

Reply:
makes sense, good reference! Would test making all rows' text aligned to the same vertical line so that the text always start at the same place in every row. but that might not look good, in which case you can ignore this 😅